Index: src/main/java/test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.io.*;\nimport java.net.*;\nimport java.util.ArrayList;\n\npublic class test {\n\n    public static void main(String[] args) throws UnknownHostException, InterruptedException {\n\n        byte[] addr = {10,1,5,77};\n        InetAddress a = InetAddress.getByAddress(addr);\n\n        System.out.println(a.getHostAddress());\n\n        ConnectionManager c = new ConnectionManager();\n        c.start();\n\n        SenderThread s = new SenderThread(a, 1234, \"OKOKmdmdmdmdm\");\n        s.start();\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/test.java b/src/main/java/test.java
--- a/src/main/java/test.java	
+++ b/src/main/java/test.java	
@@ -14,8 +14,10 @@
         ConnectionManager c = new ConnectionManager();
         c.start();
 
-        SenderThread s = new SenderThread(a, 1234, "OKOKmdmdmdmdm");
+        SenderThread s = new SenderThread(a, 1234, "THREAD PC GWEN");
         s.start();
+        SenderThread s2 = new SenderThread(a, 1234, "THREAD2 PC GWEN");
+        s2.start();
     }
 
 }
