Index: src/main/java/MainReceive.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/MainReceive.java b/src/main/java/MainReceive.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/MainReceive.java	
@@ -0,0 +1,6 @@
+public class MainReceive {
+    public static void main(String[] args) {
+        DatabaseManager db = new DatabaseManager();
+        NetworkReceiver r = new NetworkReceiver("R1");
+    }
+}
